### ISSUE TITLE
Which tasks are core?

### ISSUE THREAD

**Issue by MatthewHambley** [MetOffice/fab#295]
At the moment PSyclone is part of the core Fab. While it is core to my work it isn't really core to anyone else's build system. Some thought needs to be given to how we manage the extension of the framework with new task types.

**Comment by t00sa** [comment]
That's true but we don't need an issue for it!

**Comment by hiker** [comment]
I agree to close this, just to document this for anyone finding this issue later: the FabBase class handles this, including the ability to add new steps. It does not do a PSyclone step at all. In the `LFRic-baf` repo we designed a specific LFRic base class that adds the PSyclone step automatically.
